python/tests/unit/arcticdb/version_store/test_append.py

```diff
@@ -230,6 +230,32 @@ def test_append_out_of_order_and_sort(lmdb_version_store_ignore_order, prune_pre
             )
 
 
+@pytest.mark.parametrize("dynamic_schema", (True, False))
+@pytest.mark.parametrize("prune_previous_versions", [True, False])
+@pytest.mark.parametrize("write_sorted", [True, False])
+def test_sort_index(version_store_factory, dynamic_schema, prune_previous_versions, write_sorted):
+    lib = version_store_factory(dynamic_schema=dynamic_schema, ignore_sort_order=not write_sorted)
+    symbol = "symbol"
+
+    df_1 = pd.DataFrame(data={"col": [1, 2]}, index=[pd.Timestamp(1), pd.Timestamp(2)])
+    df_2 = pd.DataFrame(data={"col": [3, 4]}, index=[pd.Timestamp(3), pd.Timestamp(4)])
+    combined_df = pd.concat([df_1, df_2])
```

Comment:
`combined_df` is already the same as `sorted_df`?

```
In [20]: combined_df
Out[20]:
                               col
1970-01-01 00:00:00.000000001    1
1970-01-01 00:00:00.000000002    2
1970-01-01 00:00:00.000000003    3
1970-01-01 00:00:00.000000004    4

In [21]: sorted_df = combined_df.sort_index(inplace=False)

In [22]: sorted_df
Out[22]:
                               col
1970-01-01 00:00:00.000000001    1
1970-01-01 00:00:00.000000002    2
1970-01-01 00:00:00.000000003    3
1970-01-01 00:00:00.000000004    4
```

So the `sort_index` call doesn't actually change anything in this test?

Reply:
Ah sorry, my bad. Forgot to swap `df_1` and `df_2` in case `write_sorted=False`